jupyter-extension/src/plugins/mount/mountDrive.ts

```diff
@@ -2,16 +2,22 @@ import {ModelDB} from '@jupyterlab/observables';
 import {Contents, ServerConnection} from '@jupyterlab/services';
 import {PartialJSONObject} from '@lumino/coreutils';
 import {Signal, ISignal} from '@lumino/signaling';
+import {showErrorMessage} from '@jupyterlab/apputils';
 import {DocumentRegistry} from '@jupyterlab/docregistry';
 import {URLExt} from '@jupyterlab/coreutils';
 import {requestAPI} from '../../handler';
-import {Paging} from './paging';
 import {MOUNT_BROWSER_PREFIX} from './mount';
 
-// How many files to render in the FileBrowser UI per page
-const MAX_NUM_CONTENTS_PAGE = 100;
-// How many items to request per page
-const PAGINATION_NUMBER = 400;
+// How many Content li are visible at a given time for the user to scroll.
+const VISIBLE_CONTENT_LIS = 500;
```

Comment:
feel free to not do this, but maybe `VISIBLE_CONTENT_LI_COUNT` would be a clearer name?

Reply:
I like that name better will change!